go.mod

```diff
@@ -4,6 +4,10 @@ go 1.14
 
 require (
```

Comment:
Suggest you to run `go mod clean` before pushing again, just in case.

Reply:
ran "go mod tidy" . Done.